setup.py

```diff
@@ -29,7 +29,7 @@
     name="pytket-iqm",
     version=metadata["__extension_version__"],
     author="TKET development team",
-    author_email="tket-support@cambridgequantum.com",
+    author_email="tket-support@quantinuum.com",
```

Comment:
This address is not yet set up for forwarding, we have to keep the old one for now.

Reply:
Okay. I've changed it back.